packages/tocco-ui/src/EditableValue/StyledEditableValue.js

```diff
@@ -23,14 +23,15 @@ const StyledInputCss = css`
   background-color: transparent;
   border: 0;
   cursor: ${({immutable}) => immutable ? 'not-allowed' : 'default'};
+  pointer-events: ${({immutable}) => immutable ? 'none' : 'auto'};
   flex-grow: 1;
   min-height: 2.6rem;
   min-width: 0;
   outline: 0;
   padding: 0;
   ${() => declareFont({
-    color: theme.color('text')
-  })}
+  color: theme.color('text')
+})}
```

Comment:
intentionally? looks a bit off now.

Reply:
Must have been the auto format feature...looks weird to the human eye now indeed.